frontend/src/store/facilityBooking/action.ts

```diff
@@ -13,6 +13,61 @@ export const SetCreateBookingError = (newError: string) => async (dispatch: Disp
   })
 }
 
+export const updateDailyView = (date: Date, selectedFacilityId: number) => async (dispatch: Dispatch<ActionTypes>) => {
+  let updatedFB: Booking[] = []
+  const updatedTB: TimeBlock[] = defaultTimeBlocks
+
+  const adjustedStart = new Date(date.getFullYear(), date.getMonth(), date.getDate(), 0, 0, 0, 0)
+  const adjustedEnd = new Date(date.getFullYear(), date.getMonth(), date.getDate(), 23, 59, 59, 999)
+  const querySubString =
+    selectedFacilityId +
+    '/' +
+    '?startTime=' +
+    parseInt((adjustedStart.getTime() / 1000).toFixed(0)) +
+    '&endTime=' +
+    parseInt((adjustedEnd.getTime() / 1000).toFixed(0))
+
+  resetTB()
+  fetch(DOMAIN_URL.FACILITY + ENDPOINTS.FACILITY_BOOKING + '/' + querySubString, {
+    method: 'GET',
+    mode: 'cors',
+  })
+    .then((resp) => resp.json())
+    .then(async (res) => {
```

Comment:
is there a need to use async for this?

Reply:
fixed in [dd8ee3e](https://github.com/rhdevs/rhapp/pull/628/commits/dd8ee3e66243b7621b53856d8bbc7980e506b869)